seventh-lesson/src/taskInterfaces/robot/RobotRunner.java

```diff
@@ -0,0 +1,41 @@
+package taskInterfaces.robot;
+
+import taskInterfaces.robot.hands.Hand;
+import taskInterfaces.robot.heads.Head;
+import taskInterfaces.robot.legs.Leg;
+
+public class RobotRunner {
+    public static void main(String[] args) {
+        Robot r1 = new Robot(new Leg("Sony", 2000),
+                new Hand("Toshiba", 2630),
```

Comment:
Please create more specific robot parts:
ToshibaHead, price of it could be specified already in class.
In such case you robot construction will be
robot = new Robot(new SonyLeg(), new ToshibaHead(), new SamsungHand())
as price is already specified in parts you should have getPrice in parts and call them from robot and get final price and print it.